inngest/src/main/kotlin/com/inngest/Comm.kt

```diff
@@ -81,7 +81,13 @@ class CommHandler(
 
             val result = function.call(ctx = ctx, client = client, requestBody)
             var body: Any? = null
-            if (result.statusCode == ResultStatusCode.StepComplete || result is StepOptions) {
+            if (result.statusCode in
+                setOf(
+                    ResultStatusCode.StepComplete,
+                    ResultStatusCode.StepError,
```

Comment:
wonder if it's worth pulling out this set as a constant, maybe calling it something like "terminal" status codes? Fine to leave as is if other SDKs aren't doing that